kubernetes/__init__.py

```diff
@@ -14,7 +14,7 @@
 
 __project__ = 'kubernetes'
 # The version is auto-updated. Please do not edit.
-__version__ = "17.0.0-snapshot"
```

Comment:
This is wrong. You are changing the master version. For v17.17.0b1, all changes should be done in the `release-17.0` branch. Maybe you need to open a different PR and set the target branch to be `release-17.0` instead of `master`